nwnltr.c

```diff
@@ -80,20 +81,21 @@ void parse_cmdline(int argc, char *argv[]) {
 // Files can have fewer (just alpha), but there is no point as the special ones
 // can just be given a probability of 0 to achieve the same effect.
 // Thus, 28 is hardcoded here, for ease of file IO.
+#define MAX_LETTERS 28
```

Comment:
Good call. I'd in fact do 
```C
#ifndef MAX_LETTERS
#define MAX_LETTERS 28
#endif
```

so you could override it from command line with `-D`. Maybe `s/MAX_LETTERS/NUM_LETTERS/g`

Reply:
Good idea. I'll do that.